src/WinRT.Runtime/Projections/NotifyCollectionChangedEventHandler.cs

```diff
@@ -13,6 +13,7 @@ namespace ABI.System.Collections.Specialized
 
     [global::System.ComponentModel.EditorBrowsable(global::System.ComponentModel.EditorBrowsableState.Never)]
     [Guid("8B0909DC-2005-5D93-BF8A-725F017BAA8D")]
+    [WuxMuxProjectedType(wuxIID: "ca10b37c-f382-4591-8557-5e24965279b0", muxIID: "8B0909DC-2005-5D93-BF8A-725F017BAA8D")]
```

Comment:
Nit: can we always use uppercase for all GUIDs for consistency? Applies to all occurrences in this PR.